README.md

```diff
@@ -1,19 +1,218 @@
 # **Allstar**
 
-GitHub App installed on organizations or repositories to set and enforce security policies.
+Allstar is a GitHub App installed on organizations or repositories to set and
+enforce security policies. It's goal is to be able to detect any GitHub setting,
+or repository file contents that may be risky or not follow security best
+practices. Then, if Allstar finds a repository to be out of compliance, it will
+take an action such as create an issue.
 
-##
-**Status**
+The specific policies are intended to be highly configurable, to try to meet the
+needs of different project communities and organizations. Also, developing and
+contributing new policies is intended to be easy as well.
 
-Currently in early prototype phase. Working on finishing code for "mvp" and documentation for running and contributing policies.
+Allstar is developed under the [OpenSSF](https://openssf.org/) and as a part of
+the [Securing Critical Projects Working
+Group](https://github.com/ossf/wg-securing-critical-projects). The OpenSSF runs
+[an instance of Allstar here](https://github.com/apps/allstar-app) for anyone to
+install and use on their GitHub organizations. However, Allstar can be run by
+anyone if need be, see [the operator docs](operator.md) for more details.
 
-## 
-**Scope**
+## **Quick start**
 
-Security related settings/files only. Allstar is not a general GitHub management bot.
+[Install Allstar](https://github.com/apps/allstar-app) on your organizations and
+repositories like any other GitHub App. Follow the [quick start
+instructions](quick-start.md) to setup the configuration files needed to enable
+Allstar on your repositories.
 
+These quick start instructions don't provide much explanation, for more details
+on all the configuration options, continue reading.
 
-# 
-**Get Involved**
+### [Help! I'm getting issues created by Allstar and I don't want them.](opt-out.md)
 
-A part of the [Securing Critical Projects WG](https://github.com/ossf/wg-securing-critical-projects).
+## **Enable Configuration**
+
+Allstar can be enabled on individual repositories at the app level, and
```

Comment:
not sure what 'app level' means. individual repository level?

Reply:
The Allstar app can be enabled/disabled overall, and then individual policies can be enabled/disabled.